Packs/SecureWorks/Integrations/TaegisXDR/TaegisXDR.py

```diff
@@ -194,67 +197,111 @@ def execute_playbook_command(client: Client, env: str, args=None):
+                    resolution_reason
+                    attack_technique_ids
+                    entities{
+                      entities
+                      relationships{
+                        from_entity
+                        relationship
+                        to_entity
+                      }
+                    }
+                    metadata {
+                      engine {
+                        name
+                      }
+                      creator {
+                        detector {
+                          version
+                          detector_id
+                        }
+                        rule {
+                          rule_id
+                          version
+                        }
+                      }
+                      title
+                      description
+                      confidence
+                      severity
+                      created_at {
+                        seconds
+                      }
+                    }
+                    investigation_ids {
+                      id
+                    }
+                    sensor_types
+                }
             }
-            investigations
-            source {
-                uuid
-                origin
-                source_event
-                event_snippet
+        """
+
+    if args.get("ids"):
+        field = "alertsServiceRetrieveAlertsById"
+        query = """
+        query alertsServiceRetrieveAlertsById {
+            alertsServiceRetrieveAlertsById(
+                in: {
+                    iDs: %s
+                }
+            ) {
+                %s
             }
-            related_entities
-            references {
-                description
-                url
+        }
+        """ % (str(args["ids"]), fields)
+    else:
+        field = "alertsServiceSearch"
+        query = """
+        query alertsServiceSearch($cql_query: String, $limit: Int, $offset: Int) {
+            alertsServiceSearch(
+                in: {
+                    cql_query:$cql_query,
+                    offset:$offset,
+                    limit:$limit
+                }
+            ) {
+                %s
             }
         }
-    }
-    """ % (str(args["ids"]))
+        """ % (fields)
 
-    result = client.graphql_run(query)
-    readable_output = f'## Results\nFound {len(result["data"]["alerts"])} alerts'
+    result = client.graphql_run(query=query, variables=variables)
+    alerts = result["data"][field]["alerts"]["list"]
 
-    if result["data"]["alerts"]:
-        readable_output += "\n\n### Alerts\n"
-        for alert in result["data"]["alerts"]:
-            readable_output += f"* [{alert['message']}]({ENV_URLS[env]['xdr']}/alerts/{alert['id']})\n"
+    readable_output = f'## Results\nFound {len(alerts)} alerts'
 
-    outputs = result["data"]["alerts"]
+    if alerts:
+        readable_output += "\n\n### Alerts\n"
+        for alert in alerts:
```

Comment:
Have you consider using the TableToMarkdown function?
you can check https://xsoar.pan.dev/docs/integrations/code-conventions#tabletomarkdown for more info.

Reply:
I haven't, although that does seem useful. If possible, I'm looking to make a few more smaller changes to the integration in a few weeks and would like to include this with those changes but I'd like to get this update completed first due to the current integration being broken from our Alerts v1 API decom.